pkg/tools/sandbox/types.go

```diff
@@ -39,7 +39,7 @@ type SupplyNSMgrProxyFunc func(context.Context, string, networkservice.NetworkSe
 type SupplyNSMgrFunc func(context.Context, *registryapi.NetworkServiceEndpoint, networkservice.NetworkServiceServer, token.GeneratorFunc, grpc.ClientConnInterface, ...grpc.DialOption) nsmgr.Nsmgr
 
 // SupplyRegistryFunc supplies Registry
-type SupplyRegistryFunc func(ctx context.Context, expiryDuration time.Duration, proxyRegistryURL *url.URL, options ...grpc.DialOption) registry.Registry
+type SupplyRegistryFunc func(ctx context.Context, proxyRegistryURL *url.URL, options ...memory.Option) registry.Registry
```

Comment:
This change is critical because it can be used for testing registry-k8s or any other registry backend. So we should not use only memory.Option.